pkgs/development/tools/pip-tools.nix

```diff
@@ -1,16 +1,25 @@
-{ stdenv, buildPythonPackage, pythonPackages }:
-buildPythonPackage rec {
-  pname = "pip-tools";
+{ stdenv, glibcLocales, fetchFromGitHub, pythonPackages }:
+
+pythonPackages.buildPythonApplication rec {
   version = "1.8.1rc3";
-  name = "${pname}-${version}";
+  name = "pip-tools-${version}";
 
-  src = pythonPackages.fetchPypi {
-    inherit pname version;
-    sha256 = "0np4zla3jvxw1fpdynkwl6071j810ykbir949c2j5i2qy0a7ay4i";
+  src = fetchFromGitHub {
+    owner = "jazzband";
+    repo = "pip-tools";
+    rev = version;
+    sha256 = "09rbgzj71bfp1x1jfr1zx3vax4qjbw5l6vcd3fqvshsdvg9lcnpx";
```

Comment:
likely because the archive does not contain the tests. See the title of the commit.